samples/tests/package.json

```diff
@@ -20,7 +20,7 @@
   "license": "MIT",
   "dependencies": {
     "@azure/communication-react": "1.0.1-beta.2",
```

Comment:
if this uses `@azure/communication-react` this should likely also specify the `@azure/communication-calling` and `chat` deps like the static html sample